packages/connect-iframe/src/sharedLoggerWorker.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference lib="webworker" />
+
+const ports = [];
+const messages = [];
+
+// Debug it:
+// chrome://inspect/#workers
+
+// TODOs:
+// - Make loader work with TS syntax, for some reason it does not now.
```

Comment:
in transport-refactor-2 branch I was able to have support for sharedworker types. not sure what was to magic though

Reply:
Yes, I saw it is working in transport. I guess some detail configuration in webpack would be required.
I did not give much importance to it for now just to make it work and move forward.